Cannot build 'next' branch Axum Webhook example    (*w/ Axum >= 0.8.0 due to #[async_trait] removal. easy fix. remove attribute and use statement)

### Describe the bug

Attempting to build this example :
https://github.com/arlyon/async-stripe/blob/8c872d15681b7c375b0dd23ca7c4cacffc666882/examples/webhook-axum.rs#L50

error[E0195]: lifetime parameters or bounds on associated function `from_request` do not match the trait declaration

         async fn from_request(req: axum::http::Request<Body>, state: &S) -> Result<Self, Self::Rejection> {
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ lifetimes do not match associated function in trait


### To Reproduce

1. Try to build 'next' branch webhook example:
  https://github.com/arlyon/async-stripe/blob/next/examples/webhook-axum/src/main.rs

2. See error

### Expected behavior

Successful build...

### Code snippets

```rust

```

### OS

Linux

### Rust version

rustc 1.87.0

### Library version

1.0.0-alpha.2

### API version

 2016-07-06

### Additional context

(not actually sure about my API version, this is the Default API version in my Stripe Dashboard)

--- a/examples/webhook-axum/Cargo.toml
+++ b/examples/webhook-axum/Cargo.toml
@@ -6,6 +6,7 @@ publish = false
 
 [dependencies]
 tokio = { version = "1.24.1", features = ["rt-multi-thread", "macros"] }
-axum = { version = "0.7.4", features = ["macros"] }
-async-stripe-webhook = { path = "../../async-stripe-webhook", features = ["async-stripe-checkout"] }
-async-trait = "0.1"
+axum = { version = "0.8.4", features = ["macros"] }
+async-stripe-webhook = { path = "../../async-stripe-webhook", features = [
+    "async-stripe-checkout",
+] }

--- a/examples/webhook-axum/src/main.rs
+++ b/examples/webhook-axum/src/main.rs
@@ -13,7 +13,6 @@
 //! ```
 
 use axum::{
-    async_trait,
     body::Body,
     extract::FromRequest,
     http::{Request, StatusCode},
@@ -39,7 +38,6 @@ async fn main() {
 
 struct StripeEvent(Event);
 
-#[async_trait]
 impl<S> FromRequest<S> for StripeEvent
 where
     String: FromRequest<S>,

